2021/ml4co-competition/index.html

```diff
@@ -654,17 +646,16 @@ <h2>Datasets</h2>
 				(MILP) instances in the standard
 				<a href="https://en.wikipedia.org/wiki/MPS_(format)">MPS file format</a>.
 				Each benchmark is
-				split into <b>a training and a test set</b>, which originate from
+				split into <b>a training and a test set</b>, resulting from
 				the same problem distribution. While the training instances
 				are	made public at the beginning of the competition for
-				participants to train their models, the test instances will
-				be kept hidden for evaluation purposes, and will
-				only be revealed at the end of the competition.
+				participants to train their models. The test instances will
+				be kept hidden for evaluation purposes and will
```

Comment:
Sounds weird. Either 'While' has to be removed, or the two parts should be in the same sentence.